providers/GroupFiltersProvider.js

```diff
@@ -10,7 +10,7 @@ const GroupFiltersProviderDispatchContext = createContext();
 // Frozen to convey that these are static.
 const options = Object.freeze({
   campuses: [],
-  days: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
+  days: ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'],
```

Comment:
Sounds like we need to fight about the starting day of the week 😏

Reply:
@drewbarontini it is a church 'Sun' is always the most important day. 😝

Reply:
@drewbarontini 🥊 😆 🥊

Yeah, like @jordanwade said I figure in context of church schedule/calendar, Sunday would be the `0` day.
Having it last feels like a subtle downplay of importance.

This was an impulsive decision I can easily be persuaded off of though.
I thought the prototype started with **Sun** but looks like I'm wrong!

<img src="https://user-images.githubusercontent.com/1812955/107286081-1b0feb00-6a2e-11eb-830c-2e2043e2aa24.png" width="384" />